x/nodes/types/params.go

```diff
@@ -91,28 +103,36 @@ func (p *Params) ParamSetPairs() sdk.ParamSetPairs {
 		{Key: KeyRelaysToTokensMultiplier, Value: &p.RelaysToTokensMultiplier},
 		{Key: KeyMaxChains, Value: &p.MaximumChains},
 		{Key: KeyMaxJailedBlocks, Value: &p.MaxJailedBlocks},
+		{Key: KeyServicerStakeFloorMultiplier, Value: &p.ServicerStakeFloorMultiplier},
+		{Key: KeyServicerStakeWeightMultiplier, Value: &p.ServicerStakeWeightMultiplier},
+		{Key: KeyServicerStakeWeightCeiling, Value: &p.ServicerStakeWeightCeiling},
+		{Key: KeyServicerStakeFloorMultiplierExponent, Value: &p.ServicerStakeFloorMultiplierExponent},
 	}
 }
 
 // DefaultParams returns a default set of parameters.
 func DefaultParams() Params {
 	return Params{
-		UnstakingTime:            DefaultUnstakingTime,
-		MaxValidators:            DefaultMaxValidators,
-		StakeMinimum:             DefaultMinStake,
-		StakeDenom:               sdk.DefaultStakeDenom,
-		MaxEvidenceAge:           DefaultMaxEvidenceAge,
-		SignedBlocksWindow:       DefaultSignedBlocksWindow,
-		MinSignedPerWindow:       DefaultMinSignedPerWindow,
-		DowntimeJailDuration:     DefaultDowntimeJailDuration,
-		SlashFractionDoubleSign:  DefaultSlashFractionDoubleSign,
-		SlashFractionDowntime:    DefaultSlashFractionDowntime,
-		SessionBlockFrequency:    DefaultSessionBlocktime,
-		DAOAllocation:            DefaultDAOAllocation,
-		ProposerAllocation:       DefaultProposerAllocation,
-		RelaysToTokensMultiplier: DefaultRelaysToTokensMultiplier,
-		MaximumChains:            DefaultMaxChains,
-		MaxJailedBlocks:          DefaultMaxJailedBlocks,
+		UnstakingTime:                        DefaultUnstakingTime,
+		MaxValidators:                        DefaultMaxValidators,
+		StakeMinimum:                         DefaultMinStake,
+		StakeDenom:                           sdk.DefaultStakeDenom,
+		MaxEvidenceAge:                       DefaultMaxEvidenceAge,
+		SignedBlocksWindow:                   DefaultSignedBlocksWindow,
+		MinSignedPerWindow:                   DefaultMinSignedPerWindow,
+		DowntimeJailDuration:                 DefaultDowntimeJailDuration,
+		SlashFractionDoubleSign:              DefaultSlashFractionDoubleSign,
+		SlashFractionDowntime:                DefaultSlashFractionDowntime,
+		SessionBlockFrequency:                DefaultSessionBlocktime,
+		DAOAllocation:                        DefaultDAOAllocation,
+		ProposerAllocation:                   DefaultProposerAllocation,
+		RelaysToTokensMultiplier:             DefaultRelaysToTokensMultiplier,
+		MaximumChains:                        DefaultMaxChains,
+		MaxJailedBlocks:                      DefaultMaxJailedBlocks,
+		ServicerStakeFloorMultiplier:         DefaultServicerStakeFloorMultiplier,
```

Comment:
Remove 
```
		ServicerStakeFloorMultiplier:         DefaultServicerStakeFloorMultiplier,
		ServicerStakeWeightMultiplier:        DefaultServicerStakeWeightMultiplier,
		ServicerStakeWeightCeiling:           DefaultServicerStakeWeightCeiling,
		ServicerStakeFloorMultiplierExponent: DefaultServicerStakeFloorMultiplierExponent,
``` 
From default parameters, as these parameters are not part of the original parameter set, this breaks compatibility when calling initgenesis.  Test broken : TestDefaultParams line 12 x/nodes/types/params_test.go

Reply:
Keep in mind that changing this may break other test that relied on default values as they all must be 0 until the feature is turned on

Reply:
Ahh ok makes sense! have removed the entries in the squashed commit